UnitsNet/Scripts/UnitDefinitions/Ratio.json

```diff
@@ -24,6 +36,26 @@
                 {
                     "Culture": "en-US",
                     "Abbreviations": ["%"]
+                },
+                {
+                    "Culture": "ru-RU",
+                    "Abbreviations": ["% открытия"]
+                }
+            ]
+        },
+	{
+            "SingularName": "ComplementPercent",
+            "PluralName": "ComplementPercent",
+            "FromUnitToBaseFunc": "1-x/1e2",
+            "FromBaseToUnitFunc": "100-x*1e2",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": ["%"]
```

Comment:
Hm, I wonder how `Ratio.Parse("5 %")` would work. The case with multiple units sharing the exact same abbreviation is not explored, at least not by me.

I would expect the above statement to equal `Ratio.FromPercent(5)`, so how do we ensure that?
The easiest fix is to simply change ComplementPercent abbreviation to `% comp.` or similar. 

Thoughts?